modules/jabba/1.0/jabba.smk

```diff
@@ -0,0 +1,349 @@
+    conda: CFG["conda_envs"]["jabba"]
+    shell:
+        op.as_one_line("""
+        Rscript -e 'Sys.setenv(R_REMOTES_NO_ERRORS_FROM_WARNINGS = TRUE)' 
+                -e 'remotes::install_github("mskilab/dryclean", upgrade = FALSE)'
+            &&
+        touch {output.complete}
+        """)
+
+rule _jabba_install_jabba:
+    output:
+        complete = CFG["dirs"]["jabba"] + "jabba.installed"
+    conda: CFG["conda_envs"]["jabba"]
+    params:
+        cplex_dir = CFG["CPLEX_DIR"]
+    shell:
+        op.as_one_line(""" 
+        Rscript -e 'Sys.setenv(R_REMOTES_NO_ERRORS_FROM_WARNINGS = TRUE)'
+                -e 'Sys.setenv(CPLEX_DIR = "{params.cplex_dir}")'
+                -e 'remotes::install_github("mskilab/JaBba", upgrade = FALSE)'
+            &&
+        touch {output.complete}
+        """)
+
+# Symlinks the input files into the module results directory (under '00-inputs/')
+rule _jabba_input_bam:
+    input:
+        bam = CFG["inputs"]["sample_bam"],
+        bai = CFG["inputs"]["sample_bai"]
+    output:
+        bam = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bam",
+        bai = CFG["dirs"]["inputs"] + "bam/{seq_type}--{genome_build}/{sample_id}.bai",
+        crai = CFG["dirs"]["inputs"]+ "bam/{seq_type}--{genome_build}/{sample_id}.crai"
+    run:
+        op.relative_symlink(input.bam, output.bam)
+        op.relative_symlink(input.bai, output.bai)
+        op.relative_symlink(input.bai, output.crai)
+
+rule _jabba_input_junc_bedpe:
+    input:
+        junc = CFG["inputs"]["sample_junc"]
+    output:
+        junc = CFG["dirs"]["inputs"] + "junc/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.bedpe"
+    run:
+        op.relative_symlink(input.junc, output.junc)
+
+rule _jabba_input_junc_vcf:
+    input:
+        junc = CFG["inputs"]["sample_junc"]
+    output:
+        junc = CFG["dirs"]["inputs"] + "junc/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}.vcf"
+    run:
+        op.relative_symlink(input.junc, output.junc)
+
+# Runs fragcounter on individual samples
+rule _jabba_run_fragcounter:
+    input:
+        installed = str(rules._jabba_install_fragcounter.output.complete),
+        bam = str(rules._jabba_input_bam.output.bam),
+        gc = reference_files("genomes/{genome_build}/annotations/jabba/gc1000.rds"),
+        map = reference_files("genomes/{genome_build}/annotations/jabba/map1000.rds"),
+        run_custom_fc = CFG["inputs"]["run_custom_fc"]
+    output:
+        rds = CFG["dirs"]["fragcounter"] + "run/{seq_type}--{genome_build}/{sample_id}/cov.rds"
+    log:
+        stdout = CFG["logs"]["fragcounter"] + "run/{seq_type}--{genome_build}/{sample_id}/fc.stdout.log",
+        stderr = CFG["logs"]["fragcounter"] + "run/{seq_type}--{genome_build}/{sample_id}/fc.stderr.log"
+    conda: CFG["conda_envs"]["jabba"]
+    threads: CFG["threads"]["fragcounter"]
+    resources:
+        mem_mb = CFG["mem_mb"]["fragcounter"]
```

Comment:
in this rule and the rules after, can you please change this to resource unpacking?